### ISSUE TITLE
Wrong Accuracy value for Training data in tutorial...

### ISSUE THREAD

**Issue by ManishAradwad** [tensorflow/tensorflow#35249]
## URL(s) with the issue:
https://www.tensorflow.org/tutorials/keras/classification/

## Description of issue (what needs changing):

Under "Train the model" in "Build the model", the accuracy of the model on training data after 10 epochs is 0.91(91%) while it is mentioned as 0.88(88%).

### Clear description

Since, it is already mentioned in the tutorials that the model overfits the training data, thus the accuracy on training data should be more than that on testing data(88.3%).
### Submit a pull request?

If this issue is alright, I'll be glad the submit a PR right away...
Thanks for the help!

**Comment by ymodak** [comment]
Perhaps we may update the statement to `This model reaches an accuracy of about 0.91 (or 91%) on the training data` in the `Train the model` cell.

**Comment by ManishAradwad** [comment]
Yes, Sure! I'll make the changes right away

**Comment by ManishAradwad** [comment]
I'm sorry but I didn't get you clearly, Are you saying that we should add 
`model.fit(train_images, train_labels, epochs=10)`
and
`As the model trains, the loss and accuracy metrics are displayed. This model reaches an accuracy of about 0.91 (or 91%) on the training data.`
to the cell `Train the model` above??

**Comment by ManishAradwad** [comment]
Looks like we have 89% accuracy for 5 epochs while for 10 epochs, accuracy is 91%
So, i think instead changing the epochs, changing accuracy is fine...
This is what I've done in this PR: https://github.com/tensorflow/docs/pull/1311
Can someone plz check it out.

**Comment by ymodak** [comment]
Closing this issue since the associated PR has been merged. Thanks!
